src/components/NewsCard.jsx

```diff
@@ -0,0 +1,75 @@
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import { Container, Grid } from '@mui/material';
+import { createTheme } from '@mui/material';
+import { ThemeProvider } from '@emotion/react';
+
+const theme = createTheme({
+    typography:{
+      fontFamily: 'Poppins',
+      fontWeightLight: 200,
+      fontWeightRegular: 300,
+      fontWeightMedium: 400,
+      fontWeightBold: 500,
+    }
+  })
+
+const news = [
```

Comment:
keep this thing in seperate file. make something like data.js and export it from there

Reply:
Done